docs/services/slam/cartographer/_index.md

```diff
@@ -51,7 +51,7 @@ Select from the following modes to obtain the correct instructions to configure
 
 {{% alert title="REQUIREMENTS" color="tip" %}}
 
-Running `cartographer` in Live Data Collection mode requires a [RPLIDAR A1](https://www.slamtec.com/en/Lidar/A1) or [RPLIDAR A3](https://www.slamtec.com/en/Lidar/A3) LIDAR scanning device.
+Running `cartographer` in Live Data Collection mode requires a [RPLIDAR A1](https://www.slamtec.com/en/Lidar/A1) or [RPLIDAR A3](https://www.slamtec.com/en/Lidar/A3) LIDAR scanning device. The default cartographer settings, and the example config shown below (which uses the default cartographer settings), are for a RPLIDAR A3. See the notes next to the 'config_params' for recommended settings for RPLIDAR A1.
```

Comment:
[nit] Can you add links to the following?
* Default cartographer settings
* example config
* `config_params`

Otherwise it is difficult to follow what you're referring to.

[opt]
```suggestion
Running `cartographer` in Live Data Collection mode requires a [RPLIDAR A1](https://www.slamtec.com/en/Lidar/A1) or [RPLIDAR A3](https://www.slamtec.com/en/Lidar/A3) LIDAR scanning device. The default cartographer settings and the example config shown below (which uses the default cartographer settings) are nominal parameters one can use for an RPLIDAR A3. See the notes next to the 'config_params' for recommended settings for an RPLIDAR A1.
```

Reply:
Sounds good! Will do.

Reply:
@kkufieta I suggested this change, let me know if it looks good to you? https://github.com/viamrobotics/docs/pull/1097#discussion_r1175753033

Reply:
Looks good to me! :)